app/controllers/reviews_controller.rb

```diff
@@ -5,17 +5,23 @@ def index
   end
 
   def new
-    @burger = Burger.find(params[:burger_id])
-    @review = Review.new
+    if current_user.present?
```

Comment:
I think you'll find that `Devise` provides a helper to do this exact thing, called `user_signed_in?`.